subsys/net/lib/sockets/sockets_tls.c

```diff
@@ -64,6 +64,9 @@ struct tls_context {
 	/** Socket flags passed to a socket call. */
```

Comment:
Typo in commit message of "net: tls: Support DTLS send and handshake".
s/failes/fails/